src/containers/RepositoriesContainer/RepositoriesContainer.js

```diff
@@ -0,0 +1,45 @@
+import React, { PropTypes, Component } from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+
+// Actions
+import * as actionCreators from './RepositoriesDucks.js';
+
+// Components
+import RepoList from '../../components/RepoList/';
+
+const mapStateToProps = (state) => ({
+  isLoading: state.repositories.isLoading,
+  repos: state.repositories.list
+});
+
+const mapDispatchToProps = (dispatch) => bindActionCreators(actionCreators, dispatch);
+
+class RepositoriesContainer extends Component {
+
+  static propTypes = {
+    isLoading: PropTypes.bool.isRequired,
+    repos: PropTypes.array,
+    load: PropTypes.func.isRequired
+  };
+
+  constructor (props) {
+    super(props);
+
+    if (typeof props.repos === 'undefined') {
```

Comment:
Getting rid of it in the next PR by moving this condition to the action creator. 

Constructor will simply call the props.load() then.